packages/grid/src/BaseCell.css

```diff
@@ -0,0 +1,59 @@
+.uitkGridBaseCell {
+  box-sizing: border-box;
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+
+  border-style: solid;
+  border-top-color: var(--row-border-color);
```

Comment:
these need to start with --grid (see #380)

Reply:
changed all --uitkGrid... to --grid..., all css vars start with --grid